### PR TITLE
Adding circle_detection and interactive_face packages

### PR DESCRIPTION
**Pull request by Jailander**

### CODE REVIEW

File: circle_detection/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package scitos_docking
```
Comment (marc-hanheide) [inline circle_detection/CHANGELOG.rst:2]: `scitos_docking` is a bit of a surprise in here ;-)

**Comment by marc-hanheide** [comment]
Thanks @pet1330 and @Jailander for taking action and preparing for tomorrow... Sorry, I was absence today, I meant to discuss things with you, but we'll meet tomorrow.

**Comment by Jailander** [comment]
yes don't worry, things here are ok we have the robot moving with joystick and keyboard face and circle detector running on the robot we also moved the robot to its new position  (I'll try Moveit! on the robot tomorrow morning, it works on simulation already maybe we can show it tomorrow too). Do you thing something is missing?

**Comment by marc-hanheide** [comment]
that's brill! Much more than I expected!

**Comment by Jailander** [comment]
I correct moveit! works on the robot too ;)

**Comment by marc-hanheide** [comment]
:+1:

File: circle_detection/CHANGELOG.rst (hunk 1 above)
Comment (pet1330) [inline circle_detection/CHANGELOG.rst:2]: This needs to be cleaned up, it was just coped from my repo (which I also need to clean up) :stuck_out_tongue:

**Pull request merged**